pkg/cloudclient/aws/private.go

```diff
@@ -868,17 +873,29 @@ func (ac *Client) removeLoadBalancerFromMasterNodes(ctx context.Context, kclient
 	var intDNSName, intHostedZoneID, lbName string
 	for _, networkLoadBalancer := range nlbs {
 		if networkLoadBalancer.scheme == "internet-facing" {
+
 			lbName = networkLoadBalancer.loadBalancerName
-			err := ac.deleteExternalLoadBalancer(networkLoadBalancer.loadBalancerArn)
+
+			canDelete, err := ac.canDeleteNlb(networkLoadBalancer, clusterName)
 			if err != nil {
+				fmt.Printf("Error: Problem while trying to Delete NLB %v: %v \n", networkLoadBalancer.loadBalancerName, err)
```

Comment:
I think it might be better to stick with using `log` calls instead of `fmt` - same as in the rest of the file.

Reply:
Done